R/auto.R

```diff
@@ -64,10 +64,12 @@ pre_release_impl <- function(which, force) {
   # push main branch, bump to devel version and push again
   push_to_new(remote_name, force)
   switch_branch(main_branch)
+  push_head(main_branch)
```

Comment:
not sure if these two `push_head()` calls are still needed - guess we need to check in action